packages/fast-components-styles-msft/src/paragraph/index.ts

```diff
@@ -4,16 +4,20 @@ import { ParagraphClassNameContract } from "@microsoft/fast-components-class-nam
 import { fontWeight } from "../utilities/fonts";
 
 const styles: ComponentStyles<ParagraphClassNameContract, DesignSystem> = {
-    paragraph: {},
-    paragraph__1: {
-        fontWeight: `${fontWeight.semilight}`,
```

Comment:
I think you should actually leave these as-is, because otherwise they will have the oppertunity to override Heading

Reply:
Sorry thought this was Typography - you're good.